Sources/SwiftInspectorVisitors/Tests/StructVisitorSpec.swift

```diff
@@ -1,413 +0,0 @@
-                && $0.parentType?.asSource == "FooStruct.FooFooStruct"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.BarFooClass") {
-            let matching = self.sut.innerClasses.filter {
-              $0.name == "BarFooClass"
-                && $0.inheritsFromTypes.map { $0.asSource } == ["Equatable"]
-                && $0.parentType?.asSource == "FooStruct"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.BarFooClass.BarBarFooStruct") {
-            let matching = self.sut.structs.filter {
-              $0.name == "BarBarFooStruct"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooStruct.BarFooClass"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.BarFooEnum") {
-            let matching = self.sut.innerEnums.filter {
-              $0.name == "BarFooEnum"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooStruct"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooStruct.BarFooEnum.BarBarFooStruct") {
-            let matching = self.sut.structs.filter {
-              $0.name == "BarBarFooStruct"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooStruct.BarFooEnum"
-            }
-
-            expect(matching.count) == 1
-          }
-        }
-      }
-
-      context("visiting a code block with multiple top-level declarations") {
-        context("with multiple top-level structs") {
-          it("asserts") {
-            let content = """
-            public struct FooStruct {}
-            public struct BarStruct {}
-            """
-
-            // The StructVisitor is only meant to be used over a single struct.
-            // Using a StructVisitor over a block that has multiple top-level
-            // structs is API misuse.
-            expect(try VisitorExecutor.walkVisitor(
-                    self.sut,
-                    overContent: content))
-              .to(throwAssertion())
-          }
-        }
-
-        context("with a top-level class after a top-level struct") {
-          it("asserts") {
-            let content = """
-            public struct FooStruct {}
-            public class FooClass {}
-            """
-
-            // The StructVisitor is only meant to be used over a single struct.
-            // Using a StructVisitor over a block that has a top-level class
-            // is API misuse.
-            expect(try VisitorExecutor.walkVisitor(
-                    self.sut,
-                    overContent: content))
-              .to(throwAssertion())
-          }
-        }
-
-        context("with a top-level enum after a top-level struct") {
```

Comment:
This context was brought into the new file under a context named `with a top-level enum declaration`